rootfs/opt/fluentd/sbin/boot

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
-FLUENTD_CONF="/opt/fluentd/conf/fluentd.conf"
+FLUENTD_CONF=${FLUENTD_CONF:-"/opt/fluentd/conf/fluentd.conf"}
+INSTALL_BUILD_TOOLS=${INSTALL_BUILD_TOOLS:-"false"}
+BUILD_TOOLS='g++ gcc make ruby-dev'
```

Comment:
I would install `build-essential` instead, see https://github.com/deis/docker-go-dev/pull/60 and https://github.com/deis/docker-go-dev/pull/58 (has more details) for my explanation on that - It would replace `g++ gcc make` (as in it depends on that and more build tools)